### PR TITLE
Add checkbox story

### PR DESCRIPTION
**Pull request by maya**
Part of #1696

### CODE REVIEW

File: frontend/src/stories/Checkboxes.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+} as Meta;
+
+type Props = React.ComponentProps<typeof Checkboxes>;
+
+const Template: Story<Props> = (args) => {
+  const [fruit, setFruit] = useState("");
+
+  return (
+    <div className="grid-container margin-top-2">
+      <Checkboxes {...args} onChange={() => setFruit(fruit)} />
```
Comment (maya) [inline frontend/src/stories/Checkboxes.stories.tsx:45]: What's the correct way to implement the `onChange` so the checkbox is clickable?

Comment (cdevent) [reply frontend/src/stories/Checkboxes.stories.tsx:45]: Let me look on it.

Comment (cdevent) [reply frontend/src/stories/Checkboxes.stories.tsx]: pushed a way how it can be done.

Comment (maya) [reply frontend/src/stories/Checkboxes.stories.tsx]: thanks! that works

**Review by cdevent — APPROVED** [review]
LGTM

**Review by emmastephenson — APPROVED** [review]

**Pull request merged**